Sources/PrimerSDK/Classes/Core/Analytics/AnalyticsEvent.swift

```diff
@@ -35,32 +35,26 @@ extension Analytics.Event {
 
             var issuerId: String?
             var paymentMethodType: String?
+            var cardNetworks: [String]?
             var url: String?
             var iPay88PaymentMethodId: String?
             var iPay88ActionType: String?
 
             init(
                 issuerId: String? = nil,
                 paymentMethodType: String? = nil,
+                cardNetworks: [String]? = nil,
                 url: String? = nil,
                 iPay88PaymentMethodId: String? = nil,
                 iPay88ActionType: String? = nil
             ) {
                 self.issuerId = issuerId
                 self.paymentMethodType = paymentMethodType
+                self.cardNetworks = cardNetworks
                 self.url = url
                 self.iPay88PaymentMethodId = iPay88PaymentMethodId
                 self.iPay88ActionType = iPay88ActionType
             }
-
-            func encode(to encoder: Encoder) throws {
```

Comment:
TODO: check that this actually synthesizes to `encodeIfPresent` calls rather than encoding null JSON values